src/sagemaker_containers/_mapping.py

```diff
@@ -93,7 +93,7 @@ def arg_value(value):
     return [item for item in itertools.chain.from_iterable(items)]
 
 
-def _decode(obj):  # type: (bytes or str or unicode or object) -> unicode
+def _decode(obj):  # type: (bytes or str or unicode or object) -> unicode # noqa ignore=F821
```

Comment:
I'd remove `unicode` like with above. it's only valid in Python 2, which doesn't check these annotations

Reply:
i wanted to keep this function mentioning it since it's explicitly differentiate  between python2 unicode and python3 strings, for humans, not for flake8:)

Reply:
gotcha, fair enough